graph-builder/src/registry.rs

```diff
@@ -108,6 +109,8 @@ pub fn fetch_releases(
         .build()
         .map_err(|e| format_err!("{}", e))?;
 
+    let quay_client: quay::v1::Client = quay::v1::Client::builder().build()?;
```

Comment:
This probably needs an access token to be set on the builder, coming from config. Were you planning to add it later?